src/test/java/org/skypro/exam/ExaminerServiceImplTest.java

```diff
@@ -0,0 +1,53 @@
+package org.skypro.exam;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.skypro.exam.model.Question;
+import org.skypro.exam.service.ExaminerServiceImpl;
+import org.skypro.exam.service.JavaQuestionService;
+import org.skypro.exam.service.QuestionService;
+import org.skypro.exam.service.ReqestException;
+
+import java.util.Set;
+
+@ExtendWith(MockitoExtension.class)
+public class ExaminerServiceImplTest {
+    @Mock
+    private Question question;
+
+    @Mock
+    private JavaQuestionService javaQuestionService;
```

Comment:
используем плюсы полиморфизма
```suggestion
    private QuestionService javaQuestionService;
```